actix-web-codegen/src/route.rs

```diff
@@ -221,6 +224,20 @@ impl Route {
         let ast: syn::ItemFn = syn::parse(input)?;
         let name = ast.sig.ident.clone();
 
+        // Try and pull out the doc comments so that we can reapply them to the
+        // generated struct.
+        //
+        // Note that multi line doc comments are converted to multiple doc
+        // attributes.
+        let mut doc_attribute = Vec::new();
```

Comment:
```suggestion
        let mut doc_attributes = Vec::new();
```

then the return value can also use struct init shorthand for this.

Reply:
Actually I think this + the whole loop can be simplified to

```rust
let doc_attributes = ast.attrs.iter().filter(|attr| attr.path.is_ident("doc")).collect();
```